targets/TARGET_Cypress/TARGET_PSOC6/cy_usb_phy.cpp

```diff
@@ -176,6 +181,10 @@ void USBPhyHw::usb_dev_execute_ep_callbacks(void)
 void USBPhyHw::deinit()
 {
     cyhal_usb_dev_free(&obj);
+
+    if (this->events != NULL) {
+        sleep_manager_unlock_deep_sleep();
```

Comment:
deinit() should also set `events` to NULL.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/d1683c4961645a2ffd1e82700ab12d87e6afdcd1..7e0b56fa0c2c4378b88479dd2c02640e10a3da4c).